services/web/server/src/simcore_service_webserver/director_v2/_models.py

```diff
@@ -17,6 +18,12 @@ class ClusterPing(BaseModel):
     authentication: ExternalClusterAuthentication
 
 
+_DEFAULT_THUMBNAILS = {
+    f"{ClusterTypeInModel.AWS}": "https://upload.wikimedia.org/wikipedia/commons/thumb/9/93/Amazon_Web_Services_Logo.svg/250px-Amazon_Web_Services_Logo.svg.png",
+    f"{ClusterTypeInModel.ON_PREMISE}": "https://upload.wikimedia.org/wikipedia/commons/thumb/a/ac/Crystal_Clear_app_network_local.png/120px-Crystal_Clear_app_network_local.png",
+}
```

Comment:
At some point these should be backed up in a more relibale place. Could you open an issue for this?
Some GitHub repo maybe, where we have access?
I would like to avoid having broken images because the url changed.

Reply:
these as wikipedia images so I do not think that would be a problem.
Nonetheless, we already already have https://github.com/ITISFoundation/osparc-assets for those